src/index.js

```diff
@@ -166,6 +183,17 @@ const App = () => {
         </a>
       </div>`
 
+  const renderDeletePlaylistButton = () =>
+    playList.length > 0 && html`
+      <button
+        class="${css.dangerButton} cursor-pointer flex items-center content-evenly"
+        id="generated-url-div"
+        onClick=${() => openDeletePlaylistPopUp()}
+      >
+        Delete playlist
+        <img style="margin-left:50px" class="ml-1 w-5 opacity-50" src=${trashIcon} />
```

Comment:
Can you remove the `style="margin-left:50px"`? You are already using a margin-left class `ml-1`, I think `ml-1` or `ml-2` is fine. One last thing(for now) woud be put the icone before the text, then if everything is okay I will merge you PR. Thanks